modules/users/tests/server/user.server.routes.tests.js

```diff
@@ -262,6 +262,31 @@ describe('User CRUD tests', function () {
     });
   });
 
+  it('should be able to change profile picture if signed in', function (done) {
+    agent.post('/api/auth/signin')
+      .send(credentials)
+      .expect(200)
+      .end(function (signinErr, signinRes) {
+        // Handle signin error
+        if (signinErr) {
+          return done(signinErr);
+        }
+
+        agent.post('/api/users/picture')
+          .attach('newProfilePicture', './modules/users/client/img/profile/default.png')
+          .send(credentials)
+          .expect(200)
+          .end(function (changePictureErr, changePictureRes) {
+            // Handle change profile picture error
+            if (changePictureErr) {
```

Comment:
When is an upload error, the message is "Error occurred while uploading profile picture", so is a string.
Otherwise, is an user error when saving.